[2024-05-16T13:51:20.876+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-16T13:51:20.928+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: randomuser_ingestion.store_user_group_b manual__2024-05-16T13:31:32.211160+00:00 [queued]>
[2024-05-16T13:51:20.948+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: randomuser_ingestion.store_user_group_b manual__2024-05-16T13:31:32.211160+00:00 [queued]>
[2024-05-16T13:51:20.949+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 3
[2024-05-16T13:51:20.973+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): store_user_group_b> on 2024-05-16 13:31:32.211160+00:00
[2024-05-16T13:51:20.983+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3728) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-16T13:51:20.990+0000] {standard_task_runner.py:63} INFO - Started process 3730 to run task
[2024-05-16T13:51:20.985+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'randomuser_ingestion', 'store_user_group_b', 'manual__2024-05-16T13:31:32.211160+00:00', '--job-id', '1127', '--raw', '--subdir', 'DAGS_FOLDER/randomuser_ingestion.py', '--cfg-path', '/tmp/tmpkfdmpuue']
[2024-05-16T13:51:20.999+0000] {standard_task_runner.py:91} INFO - Job 1127: Subtask store_user_group_b
[2024-05-16T13:51:21.141+0000] {task_command.py:426} INFO - Running <TaskInstance: randomuser_ingestion.store_user_group_b manual__2024-05-16T13:31:32.211160+00:00 [running]> on host 8d1d9b6df744
[2024-05-16T13:51:21.331+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='randomuser_ingestion' AIRFLOW_CTX_TASK_ID='store_user_group_b' AIRFLOW_CTX_EXECUTION_DATE='2024-05-16T13:31:32.211160+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-16T13:31:32.211160+00:00'
[2024-05-16T13:51:21.334+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-16T13:51:21.354+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-16T13:51:21.355+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-16T13:51:21.368+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=randomuser_ingestion, task_id=store_user_group_b, run_id=manual__2024-05-16T13:31:32.211160+00:00, execution_date=20240516T133132, start_date=20240516T135120, end_date=20240516T135121
[2024-05-16T13:51:21.406+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-16T13:51:21.430+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-16T13:51:21.432+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
